platform/ios/src/MGLFaux3DUserLocationAnnotationView.m

```diff
@@ -49,10 +55,33 @@ - (void)update
 
     if (CLLocationCoordinate2DIsValid(self.userLocation.coordinate))
     {
-        (self.mapView.userTrackingMode == MGLUserTrackingModeFollowWithCourse) ? [self drawPuck] : [self drawDot];
-        [self updatePitch];
+        if (@available(iOS 14, *)) {
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 140000
+            if (self.mapView.locationManager.accuracyAuthorization == CLAccuracyAuthorizationFullAccuracy) {
```

Comment:
`accuracyAuthorization` is optional and PassiveLocationManager in Navigation SDK hasn't implemented this property in 1.0 release. This leads to crashes with Nav SDK 1.0 and Maps SDK 6.2.0 on iOS 14.

Reply:
mapbox/mapbox-navigation-ios#2554

Reply:
mapbox/mapbox-gl-native-ios#470